README.md

```diff
@@ -255,6 +255,45 @@ A few things to note:
   the result set twice: once when parsing the AST, and again when printing
   results
 
+## Running with Ruby version managers
+
+Referral requires Ruby version >= 2.6, but your codebase may be running on something
+older.  You have a few options for using `referral`.  You could 1) change your project's
+ruby while you run `referral` and then change it back, but this seems cumbersome and
+likely to cause annoyance.  There are better ways.
+
+## Run from outside your project's working directory
+
+If you `cd ..` from your project's working directory (assuming in that context
+you are running Ruby 2.6.x), you can run `referral` commands on your codebase by passing
+the path to that codebase to referral:
+
+```
+MyProjects $ referral MyAwesomeProject/
```

Comment:
I think the `MyProjects` and `MyAwesomeProject` faux-`PS1` is liable to confuse folks. I'd recommend trimming to the `$`